valhalla/sciapplications/views.py

```diff
@@ -216,6 +216,30 @@ def render_to_response(self, context, **kwargs):
         return pdf_response
 
 
+class SciApplicationGetPDFView(LoginRequiredMixin, DetailView):
```

Comment:
This is nearly verbatim copy and paste from the original PDF view, but there is no explanation of what it is for or what makes it different.